specification/programmableconnectivity/ProgrammableConnectivity.Management/OperatorApiConnection.tsp

```diff
@@ -125,43 +158,379 @@ union AccountType {
 }
 
 /**
- * Details about the Application that would use the Operator's Network APIs.
+ * Model defining how received data is processed.
  */
-model ApplicationProperties {
```

Comment:
It seems odd for this model to be defined in OperatorApiConnections when it's only (now) used in Gateway. Is it possible to move it?

Reply:
It has been moved to Gateway (you've commented on the removed line)